applications/luci-app-rosy-file-server/po/es/rosy-file-server.po

```diff
@@ -0,0 +1,84 @@
+msgid ""
+msgstr ""
+"Content-Type: text/plain; charset=UTF-8\n"
+"Project-Id-Version: \n"
+"POT-Creation-Date: 2019-07-15 13:12-0300\n"
+"PO-Revision-Date: 2019-08-01 22:35-0300\n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 2.2.3\n"
+"Last-Translator: Franco Castillo <castillofrancodamian@gmail.com>\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"Language: es\n"
+
+#: applications/luci-app-rosy-file-server/luasrc/model/cbi/rosy-file-server/rosy-file-server.lua:50
+msgid "<a href='%s%s'>%s</a>"
+msgstr "<a href='%s%s'>%s</a>"
```

Comment:
I think this is wrong.
Translating an URL?

Reply:
He appeared in poedit to translate